apps/condo/domains/banking/schema/BankIntegration.js

```diff
@@ -20,6 +22,11 @@ const BankIntegration = new GQLListSchema('BankIntegration', {
             template: 'banking.category.*.name',
         },
 
+        accessRights: {
+            type: Relationship,
+            ref: 'BankIntegrationAccessRight.integration',
+            many: true,
```

Comment:
I'll take a guess that m2m relations is forbidden

Reply:
We should avoid usage of m2m relations to make available further migrations to different db adapter

Reply:
Coul you please add access check only for admin for this field.